.github/ci/files/config/services.yaml

```diff
@@ -0,0 +1,87 @@
+parameters:
+    secret: ThisTokenIsNotSoSecretChangeIt
+
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    #
+    # Notification Services
+    #
+
+    Pimcore\Model\Notification\Service\NotificationService:
+        public: true
+
+    Pimcore\Model\Notification\Service\UserService: ~
```

Comment:
these can go. see https://github.com/pimcore/admin-ui-classic-bundle/pull/29/files#diff-c47b03734cd2b4337b345eeba955637ba790d51fd98979f6d366d4acbdb104e0L10

Reply:
Why was it removed. it is still used here `src/Controller/Admin/NotificationController.php` right?